packages/@aws-cdk-testing/framework-integ/test/aws-cloudformation/test/integ.import-template.ts

```diff
@@ -0,0 +1,17 @@
+import * as core from 'aws-cdk-lib';
+import * as inc from 'aws-cdk-lib/cloudformation-include';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+
+const app = new core.App();
+
+const stack = new core.Stack(app, 'ImportStack');
+
+new inc.CfnInclude(stack, 'ImportStack', {
```

Comment:
We need a test that doesn't include `CfnInclude`. We also want to test that cdk will synth and deploy a valid template.

Reply:
I am not sure what do you expected. Just a basic stack creation (with for example a sqs queue)?